addons/binding/org.openhab.binding.kostal.inverter/src/main/java/org/openhab/binding/kostal/inverter/ChannelConfig.java

```diff
@@ -0,0 +1,13 @@
+package org.openhab.binding.kostal.inverter;
+
+public class ChannelConfig {
```

Comment:
We are following this practice instead of adding every single person in the license header (which is the other approach done by many projects). It will simply show multiple author tags over time, see e.g. [here](https://github.com/eclipse/smarthome/blob/80f743bdfba10241a29689ff6cd6048a43b8b8d2/bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java#L75).